README_TEMPLATE.md

```diff
@@ -85,7 +83,7 @@ You can verify that your containers are in a running state with `docker compose
 The one tool that we don't manage through Docker Compose is `node`/`npm`. The reason is that node only needs to run on demand,
 and not be constantly run in the background while we're doing development.
 
-To build the `node`/`npm` container, run: `docker build -t project-node docker/node`
+To build the `node`/`npm` container, run: `docker build -t my-project-node docker/node`
```

Comment:
Maybe I'm being dumb here but I still think there needs to be some sort of note about that you must actually build this before you try to run docker/bin/npm - it bit me and I can see myself forgetting that again.

Reply:
Done